### ISSUE TITLE
Document "CleanupTransformer"

### ISSUE THREAD

**Issue by MikeMitterer** [dart-lang/sdk#24807]
It took me quite a while to figure out how to delete Assets with a transformer.
I took this approach: https://goo.gl/jKiQhV 
Please add a few lines of documentation how to remove!!!! Assets. - Thanks!

**Comment by kevmoo** [comment]
@nex3 Should this be moved to the barback repo? A tweak for the readme?

**Comment by nex3** [comment]
I'm not sure exactly what this is asking, but if you want to move the conversation over there, go for it.

**Comment by MikeMitterer** [comment]
When I came to this problem I first read the doc https://www.dartlang.org/tools/pub/transformers/ then I moved over to the barback documentation, found out that Barback has a function to delete assets (removeSources) search through the whole Api for a way to get the Barback-object and found finally out that there is no way to get this object. Searched through pub and github for another Transformer that does something similar (was not easy!) and finally implemented my own version.

Conclusion: Either provide a way to remove resources in a Transformer or add a sample to https://www.dartlang.org/tools/pub/transformers/#more-info 

The confusing thing at the moment is that a "CleanupTransformer" does not really delete something it avoids the creation of something.
